lib/LINE/Bot/API.pm

```diff
@@ -511,6 +512,25 @@ sub test_webhook_endpoint {
     LINE::Bot::API::Response::WebhookTest->new(%{ $res });
 }
 
+sub get_followers {
+    my ($self, $opts) = @_;
+    my $req_url = "followers/ids";
+
+    if ($opts->{'limit'} && $opts->{'start'}) {
+        $req_url .= '?limit=' . $opts->{'limit'} . "&start=" . $opts->{start};
+    } elsif ($opts->{'limit'}) {
+        $req_url .= '?limit=' . $opts->{'limit'};
+    } elsif ($opts->{'start'}) {
+        $req_url .= '?start=' . $opts->{'start'};
+    }
```

Comment:
For building URLs with query parameters, I recommend using `URI::QueryParam`, take a look at: https://github.com/line/line-bot-sdk-perl/blob/master/lib/LINE/Bot/Audience.pm#L119

You'll need to use these two statements together in order to use call `query_param` method on an URI object:

```
use URI;
use URI::QueryParam;
```

`query_param` does characater-escaping internally too, so it is obviosuly "more correct" than manually a list of query paramaters. :)